apps/userdashboard/api.py

```diff
@@ -84,6 +84,16 @@ def update(self, request, *args, **kwargs):
 
         return super().update(request, *args, **kwargs)
 
+    @action(detail=True, methods=["PATCH"])
```

Comment:
note: changed from post to patch as otherwise we have to add a rule / permission below. Also patch seems semantically more fitting?

Reply:
indeed, since we are updating and not creating it. and since we only update partially, patch makes sense.

Reply:
indeed, patch is good here since we are updating partially.
BTW are you sure default is to show to users? I don't see it in the story, and IIRC the case was that we want to prevent showing it if is non sense.

Reply:
@m4ra show default AC was in the toggle switch story which is closed